src/api.ts

```diff
@@ -143,6 +144,7 @@ export async function analyze({
               }
               reporter.addProp(componentName, searchProp, {
                 propCode: code.slice(node.start || 0, node.end || -1),
+                propValue: formatPropValue(null),
```

Comment:
I'm not sure about this propValue since I didn't find any option to test case

Reply:
Hmm, maybe this should be `undefined`? There is no prop value in this case.

Reply:
Thanks! Changed to `undefined` in 928b794